cluster/gce/gci/configure-helper.sh

```diff
@@ -2728,7 +2742,7 @@ EOF
   fi
   if [[ "${ENABLE_NODE_LOGGING:-}" == "true" ]] && \
      [[ "${LOGGING_DESTINATION:-}" == "gcp" ]]; then
-    setup-addon-manifests "addons" "fluentd-gcp"
+    setup-addon-manifests-remote "addons" "fluentd-gcp" "https://github.com/coffeepac/instrumentation-addons/tree/pc-move-fluent-gcp/fluentd-gcp"
```

Comment:
I'm a pretty strong -1 on adding remote addon installation to this script. See discussion in https://github.com/kubernetes/kubernetes/pull/101039#pullrequestreview-636921064